### DISCUSSION TITLE
Tests Showing as Failing

### DISCUSSION THREAD

**Discussion by mindyzwan** [orta/vscode-jest#969] [Q&A]
My teammate and I both downloaded the extension and we're looking at the same repo, same branch, same state. Using the CLI, the tests all pass and the extension shows them all as passing for me, but half of them are failing for my teammate. Any ideas?

**Comment by connectdotz** [comment]
hm... sounds like maybe some kind of runtime env variation? What happened after they reload window?  What error did their terminal show? console log? I assume you are running 5.1.0, right?
